restler/unit_tests/test_auth_parser.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from engine.core.request_utilities import parse_authentication_tokens, NO_TOKEN_SPECIFIED, NO_SHADOW_TOKEN_SPECIFIED
+
+class AuthParserTest(unittest.TestCase):
```

Comment:
>AuthParserTest [](http://example.com/codeflow?start=6&length=14)

Thanks for adding this!

Reply:
FYI, we do not have any end to end coverage for authentication in our CI pipeline.  It would be great if you can add a note to the PR that you also manually tested this on your real-world API example and it worked as expected.